src/pages/dataElements/form/DataElementFormFields.tsx

```diff
@@ -203,12 +190,14 @@ export function DataElementFormFields() {
                     {i18n.t('Aggregation levels')}
                 </StandardFormSectionTitle>
                 <StandardFormSectionDescription>
-                    {`
-                        @TODO(DataElementForm): Help text to describe the aggregation levels
-                          functionality. It appears as if this section hasn't been
-                          finalized yet by Joe, so I guess we'll have to talk about
-                          this particluar part.
-                    `}
+                    By default, the aggregation will start at the lowest
```

Comment:
This should be translated?

Reply:
I also think this should have a max-width. I added around `600px` on demo branch.